renderer/types/actions/info-actions.ts

```diff
@@ -1,40 +1,61 @@
-import * as consts from '../consts';
+import * as consts from '../../consts';
+
+export interface InfoPerMessageAction{
+  type: consts.InfoActionsTypes.PER_MESSAGE;
+  robotState: number;
+}
+
+export interface AnsibleDisconnectAction{
+  type: consts.InfoActionsTypes.ANSIBLE_DISCONNECT;
+}
+
+export interface RuntimeConnectAction{
+  type: consts.InfoActionsTypes.RUNTIME_CONNECT;
+}
+
+export interface MasterStatusAction{
+  type: consts.InfoActionsTypes.MASTER_ROBOT;
+}
+
+export interface RuntimeDisconnectAction{
+  type: consts.InfoActionsTypes.RUNTIME_DISCONNECT;
+}
+
+export interface UpdateCodeStatusAction{
+  type: consts.InfoActionsTypes.CODE_STATUS;
+  studentCodeStatus: string;
+}
+
+export interface IpChangeAction{
+  type: consts.InfoActionsTypes.IP_CHANGE;
+  ipAddress: string;
+}
+
+export interface NotificationChangeAction{
+  type: consts.InfoActionsTypes.NOTIFICATION_CHANGE;
+  notificationHold: number;
+}
```

Comment:
@ewc340 At line 50 in `info.ts` requires `action.notificationHold`, so I added an interface to fix the error for the moment. However there isn't an actual function(or dispatch) that handles notification change, so I am confused about how to handle this.

Reply:
@HaileyJang I see what you mean. Yea, looks like there is no `notificationChange` dispatch, but from `EditorContainer`, I do see a state slice for `notificationHold` although I'm not actually sure what this notification is in regards to. In the future, we may want to display notifications depending on features we add. For now, let's just leave it like this.